x-pack/plugins/apm/public/components/app/correlations/ml_failure_correlations.tsx

```diff
@@ -0,0 +1,410 @@
+          <EuiFlexGroup direction="column" gutterSize="none">
+            <EuiFlexItem data-test-subj="apmCorrelationsfailedTransactionsCorrelationsProgressTitle">
+              <EuiText size="xs" color="subdued">
+                <FormattedMessage
+                  data-test-subj="apmCorrelationsfailedTransactionsCorrelationsProgressTitle"
+                  id="xpack.apm.correlations.failure.progressTitle"
+                  defaultMessage="Progress: {progress}%"
+                  values={{ progress: Math.round(progress * 100) }}
+                />
+              </EuiText>
+            </EuiFlexItem>
+            <EuiFlexItem>
+              <EuiProgress
+                aria-label={i18n.translate(
+                  'xpack.apm.correlations.failure.progressAriaLabel',
+                  { defaultMessage: 'Progress' }
+                )}
+                value={Math.round(progress * 100)}
+                max={100}
+                size="m"
+              />
+            </EuiFlexItem>
+          </EuiFlexGroup>
+        </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <FailureCorrelationsHelpPopover />
+        </EuiFlexItem>
+      </EuiFlexGroup>
+      {selectedTerm ? (
+        <>
+          <EuiSpacer size="m" />
+          <Summary
+            items={[
+              <EuiBadge color="hollow">
+                {`${selectedTerm.fieldName}: ${selectedTerm.key}`}
+              </EuiBadge>,
+              `p-value: ${
+                selectedTerm.pValue !== null && selectedTerm.pValue < 0.00001
+                  ? '<.00001'
+                  : asPreciseDecimal(selectedTerm.pValue, 4)
+              }`,
+            ]}
+          />
+          <EuiSpacer size="m" />
+        </>
+      ) : null}
+      <CorrelationsTable<FailureCorrelationValue>
+        columns={errorCorrelationsColumns}
+        significantTerms={result?.response?.values}
+        status={FETCH_STATUS.SUCCESS}
+        setSelectedSignificantTerm={setSelectedSignificantTerm}
+        selectedTerm={selectedTerm}
+        onFilter={onClose}
+      />
+
+      {ccsWarning && (
+        <>
+          <EuiSpacer size="m" />
+          <EuiCallOut
+            title={i18n.translate(
+              'xpack.apm.correlations.failure.ccsWarningCalloutTitle',
+              {
+                defaultMessage: 'Cross-cluster search compatibility',
+              }
+            )}
+            color="warning"
+          >
+            <p>
+              {i18n.translate(
+                'xpack.apm.correlations.failure.ccsWarningCalloutBody',
+                {
+                  defaultMessage:
+                    'Data for the correlation analysis could not be fully retrieved. This feature is supported only for 7.14 and later versions.',
+                }
+              )}
+            </p>
+          </EuiCallOut>
+        </>
+      )}
+      <EuiSpacer size="m" />
+
+      {log.length > 0 && displayLog && (
+        <EuiAccordion
+          id="accordion1"
```

Comment:
Looks like this `id` is the one from EUI's examples, we should make it more specific with an `apm` prefix.

Reply:
Fixed here https://github.com/elastic/kibana/pull/108441/commits/155443605ee491b2eb7c2bcb7b3e458eed9591e6